services/docker-compose.yml

```diff
@@ -272,6 +272,22 @@ services:
       - default
       - interactive_services_subnet
 
+  agent:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/agent:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
+    deploy:
+      mode: global
+      endpoint_mode: dnsrr
+      resources:
+        reservations:
+          cpus: "0.5"
+          memory: "200M"
```

Comment:
I am not quite sure about these reservations. But I see below the dask-sidecar is reserving less and does more. Maybe we could reserve less and also add some limits as well? this guy should run at low frequency with low priority with low resource right?